apps/safe-claiming-app/src/widgets/ClaimingWidget.tsx

```diff
@@ -173,7 +180,14 @@ const ClaimingWidget = () => {
               >
                 Delegated to
               </Typography>
-              <SelectedDelegate delegate={currentDelegate} />
+              <Link
+                href={claimingSafeAppUrl}
+                rel="noopener noreferrer"
+                target="_blank"
+                underline="none"
+              >
+                <SelectedDelegate delegate={currentDelegate} />
```

Comment:
Same here, it would be cool to have some feedback that this is clickable similar to how `ButtonBase` provides it.

Reply:
Alternatively we could wrap the entire widget in a clickable ButtonBase which opens the claiming app.

Reply:
> Alternatively we could wrap the entire widget in a clickable ButtonBase which opens the claiming app.

That almost defeats the purpose of having buttons in the widget. I will rather provide some visual feedback that the user understands which parts of the widget are clickable.